tests/x11/yast2_snapper.pm

```diff
@@ -72,101 +31,26 @@ sub run() {
     # Start an xterm as root
     x11_start_program("xterm");
     assert_screen "xterm";
+    # Disable screen lock and blank screen for current Gnome session
+    script_run("gsettings set org.gnome.desktop.session idle-delay 0");
```

Comment:
please use single ticks to ensure variables are not expanded unintentionally in general.

Does this work also for e.g. the kde test suite or is it not triggered there? Can't currently review o3 (being down).

How about an assert_script_run instead? The potential failure should be handled explicitly.